charmcraft.yaml

```diff
@@ -26,3 +26,4 @@ parts:
       - rustc
       - cargo
       - pkg-config
+      - golang-go
```

Comment:
For most rocks we use the go snap instead of the package. One reason is that it makes it possible to select the version. Ex.

```
build-snaps:
      - go/1.22/stable
```

Reply:
yes, i've already made this change in the lib and test charms, ill remove it from here.